collect_app/src/main/java/org/odk/collect/android/widgets/QuestionWidget.java

```diff
@@ -372,4 +374,15 @@ public void stopAudio() {
             mPlayer.reset();
         }
     }
+
+    // Today clocks are almost always set one hour back or ahead.
```

Comment:
If I understand correctly, the problem is that on the day of a daylight savings transition, certain hours don't exist. For example, if the time change happens at midnight and goes forward an hour, there is no midnight on that day. The problem showed up for timezones that happen to have a change at midnight because that was the default time used. We didn't see it for more "common" timezones because they change their time at 2AM or other odd times.

If I got that right, I think it would useful to add a first sentence to this comment that explains that. For example - "Skip over a "daylight savings gap." This is needed on the day and time of a daylight savings transition because that date/time doesn't exist."